### DISCUSSION TITLE
gatherers/corosyncconf

### DISCUSSION THREAD

**Discussion by pirat013** [trento-project/wanda#113] [Q&A] [answered]
Looking into the check 6E9B82 the value 'two_node' from the corosync.conf file must be compared. 
Would that be the right fact to do the job:
			Name:     "corosync_nodes",
			Gatherer: "corosync.conf",
			Argument: "nodelist.node",
If I'm looking at the code I would say no but I'm not sure.
If it is not the right one maybe this part is missing and the same would be for 'expected_votes'.

**Comment by fabriziosestito** [comment, marked as answer]
@pirat013 
Gatherer: `corosync.conf`
Argument: `quorum.two_node`

The argument is easily inferred by checking the corosync.conf file structure 
```
...

quorum {

        # Enable and configure quorum subsystem (default: off)
        # see also corosync.conf.5 and votequorum.5
        provider: corosync_votequorum
        expected_votes: 2
        two_node: 1
}

...
```

The same goes for `expected_votes`

You can also try out the gatherers by executing this cli command:

```bash
./trento-agent facts gather --gatherer "corosync.conf" --argument "quorum.two_node"
```

**Reply by pirat013** [reply]
Thanks a lot.
